scala/scala-impl/src/org/jetbrains/plugins/scala/compiler/ScalaHydraCompilerConfigurationPanel.scala

```diff
@@ -71,15 +71,26 @@ class ScalaHydraCompilerConfigurationPanel(project: Project, settings: HydraComp
 
   def setHydraStoreDirectory(path: String): Unit = hydraStoreDirectoryField.setText(path)
 
+  val compilerRegex = """.*scala-compiler-(\d+\.\d+\.\d+)\.jar""".r
```

Comment:
I'm thinking we might want to make this regex a bit more resilient and allow for qualifiers like SNAPSHOT.